src/frontend/src/binder/expr/value.rs

```diff
@@ -250,8 +250,8 @@ mod tests {
     use crate::binder::test_utils::mock_binder;
     use crate::expr::{Expr, ExprImpl, ExprType, FunctionCall};
 
-    #[test]
-    fn test_bind_value() {
+    #[tokio::test]
+    async fn test_bind_value() {
```

Comment:
Why is async tokio test preferred?

Reply:
It's because we use SessionImpl::mock and somehow it uses an async function internally.

Reply:
Interesting ... It is a runtime error rather than compile time.